### PR TITLE
Swagger: Allowing for VMSS reference during VM creation

### PR DESCRIPTION
**Pull request by shpa-microsoft**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [ ] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

**Review by jhendrixMSFT — APPROVED** [review]

File: specification/compute/resource-manager/Microsoft.Compute/stable/2019-03-01/compute.json [hunk 1]
```diff
@@ -7258,7 +7258,11 @@
...
+        },
+        "virtualMachineScaleSet": {
+          "$ref": "#/definitions/SubResource",
+          "description": "Specifies information about the virtual machine scale set that the virtual machine should be assigned to. Virtual machines specified in the same virtual machine scale set are allocated to different nodes to maximize availability. Currently, a VM can only be added to virtual machine scale set at creation time. An existing VM cannot be added to a virtual machine scale set. <br><br>This property cannot exist along with a non-null properties.avalabilitySet reference. <br><br>Minimum api‐version: 2019‐03‐01"
```
Comment (nschonni) [inline specification/compute/resource-manager/Microsoft.Compute/stable/2019-03-01/compute.json]: avalability -> availability
```suggestion
          "description": "Specifies information about the virtual machine scale set that the virtual machine should be assigned to. Virtual machines specified in the same virtual machine scale set are allocated to different nodes to maximize availability. Currently, a VM can only be added to virtual machine scale set at creation time. An existing VM cannot be added to a virtual machine scale set. <br><br>This property cannot exist along with a non-null availabilitySet reference. <br><br>Minimum api‐version: 2019‐03‐01"
```

**Review by jhendrixMSFT — CHANGES_REQUESTED** [review]
One spelling mistake to fix.  Also can you please ACK that this functionality is live?

**Comment by shpa-microsoft** [comment]
> One spelling mistake to fix. Also can you please ACK that this functionality is live?

Fixed the typo. Also it should be made available starting aug 16th.

**Comment by jhendrixMSFT** [comment]
Awesome, please ping back when this is live so this can get merged.

**Comment by shpa-microsoft** [comment]
Hey can you please merge this change in today?

**Review by jhendrixMSFT — APPROVED** [review]

**Pull request merged**